ros_buildfarm/devel_job.py

```diff
@@ -198,6 +200,9 @@ def configure_devel_job(
     if build_file is None:
         build_files = get_source_build_files(config, rosdistro_name)
         build_file = build_files[source_build_name]
+    # Overwrite build_file.targets if build_targets is specified
+    if build_targets is not None:
+        build_file.targets = deepcopy(build_targets)
```

Comment:
I don't think the `deepcopy` is necessary here. The caller should not modify the argument after passing it anyway.